Framework/PythonInterface/mantid/plots/__init__.py

```diff
@@ -373,7 +397,9 @@ def _data_update(artists, workspace):
                 self.autoscale()
                 return container_new
 
-            return self.track_workspace_artist(args[0], plotfunctions.errorbar(self, *args, **kwargs),
+            workspace = args[0]
```

Comment:
Is intentional to leave the `spec_num` out of the `errorbar` plots? As they are associated with a single spectrum then this could be useful to keep hold of?